packages/model/keys.go

```diff
@@ -30,6 +30,10 @@ func (m *Key) Get(wallet int64) (bool, error) {
 	return isFound(DBConn.Where("id = ?", wallet).First(m))
 }
 
+func (m *Key) GetTrans(transaction *DbTransaction, wallet int64) (bool, error) {
```

Comment:
What is it? GetTransaction?

Reply:
I don't know how to name this function. It does the same as Get function but in the specified transaction.

Reply:
I have renamed it to GetWallet name.